aspnetcore/fundamentals/minimal-apis/9.0-samples/WebMinOpenApi/Program.cs

```diff
@@ -233,6 +233,41 @@ public async Task TransformAsync(OpenApiDocument document, OpenApiDocumentTransf
 // </snippet_multidoc_operationtransformer1>
 #endif
 
+#if SCHEMAtransformer1
+// <snippet_schematransformer1>
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.OpenApi;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.OpenApi.Models;
```

Comment:
Remove unused imports.
```suggestion
using Microsoft.AspNetCore.OpenApi;
```